Training24/Program.c

```diff
@@ -1,15 +1,45 @@
-// ------------------------------------------------------------------------------------
+// --------------------------------------------------------------------------------
 // Training ~ A training program for new joiners at Metamation, Batch - July 2024.
 // Copyright (c) Metamation India.
-// -------------------------------------
 //  Dhaarani Mohanasundaram
-// -------------------------------------
-// Standard Program file in Main
-// -------------------------------------
+// --------------------------------------------------------------------------------
 // Program.c
-// Program on main branch.
-// ------------------------------------------------------------------------------------
+// Program on branch A4.
+// Checks for phrase palindrome and Reverse the number then checks for palindrome.
+// --------------------------------------------------------------------------------
+#include <ctype.h>
+#include <stdbool.h>
+#include <limits.h>
 
-const char* GetMessage() {
-return "Hello, World!";
+bool CheckPhrasePalindrome (const char* phrase) {
```

Comment:
Please look up compile-time polymorphism in C. C (and most other common languages) allow you to declare function of the same name with different parameters. Also called function overloading.